sensor/kubernetes/listener/resources/convert.go

```diff
@@ -343,12 +343,12 @@ func (w *deploymentWrap) populateImageIDs(pods ...*v1.Pod) {
 }
 
 func (w *deploymentWrap) getLabelSelector(spec reflect.Value) (*metav1.LabelSelector, error) {
-	s := spec.FieldByName("Selector")
+	s := spec.FieldByName("selector")
```

Comment:
This will 100% not work. Was this an unintended extra change in a find-and-replace?

Reply:
Yes it was, and I it made a lot of tests fail. Took me a while in debugging to find this, I've already fixed it now.